Hadoop version value is hardcoded in Tez pom.xml

The pom.xml in Tez contains the hadoop.version value set as 2.7.1-SNAPSHOT 

`<hadoop.version>2.7.1-SNAPSHOT</hadoop.version>`

On ppc, this causes components which use Tez as a dependency to fail as the hadoop.version value cannot be set to the ppc version of hadoop.

For example, Oozie build failed for Hadoop dependency in Tez due to the same reason:
`[ERROR] Failed to execute goal on project oozie-sharelib-pig: Could not resolve dependencies for project org.apache.oozie:oozie-sharelib-pig:jar:4.2.0: Failed to collect dependencies at org.apache.tez:tez-mapreduce:jar:0.7.1-SNAPSHOT -> org.apache.tez:tez-api:jar:0.7.1-SNAPSHOT -> org.apache.hadoop:hadoop-aws:jar:2.7.1-SNAPSHOT: Failed to read artifact descriptor for org.apache.hadoop:hadoop-aws:jar:2.7.1-SNAPSHOT: Could not transfer artifact org.apache.hadoop:hadoop-aws:pom:2.7.1-SNAPSHOT from/to apache.snapshots.repo (https://repository.apache.org/content/groups/snapshots)`

--- a/pom.xml
+++ b/pom.xml
@@ -37,7 +37,7 @@
   <properties>
     <maven.test.redirectTestOutputToFile>true</maven.test.redirectTestOutputToFile>
     <clover.license>${user.home}/clover.license</clover.license>
-    <hadoop.version>2.7.1-SNAPSHOT</hadoop.version>
+    <hadoop.version>${hadoop.version}</hadoop.version>
     <jetty.version>6.1.26</jetty.version>
     <pig.version>0.13.0</pig.version>
     <distMgmtSnapshotsId>apache.snapshots.https</distMgmtSnapshotsId>

